src/convertToObject.js

```diff
@@ -6,7 +6,20 @@
  * @return {object}
  */
 function convertToObject(sourceString) {
-  // write your code here
+  const styleArray = sourceString
+    .split(';')
+    .map((style) => {
+      return style.trim();
+    })
+    .filter((style) => style !== '');
+
+  const styles = styleArray.reduce((prev, style) => {
+    const [name, value] = style.split(':');
```

Comment:
Be careful with splitting by ':', as it can lead to issues if the value part contains additional colons, such as in URLs or content properties.